.gitignore

```diff
@@ -60,3 +60,6 @@ build/status/MV9ubGRhc19jdWjlcy9sb2cvTkxEQVNfM*.yml
 !/6_temp_coop_fetch/in/*.ind
 !/6_temp_coop_fetch/in/*.placeholder
 
+# tmp wqp indicators
+6_temp_wqp_fetch/tmp/*
+build/status/Nl90ZW1wX3dxcF9mZXRjaC90bX*
```

Comment:
I thought you were the one who had decided that it didn't work? But regardless I figured I'd try it one more time myself to see if I could determine whether it was doing what I needed. Does seem to be ignoring the right files at least on the scale of a few test files.

I think what name mangling does is map each alpha-numeric-punct character into one or more alphanumeric characters, so it seems to me that this approach should be possible. Because I don't have a key for the mapping, I've just been comparing filenames to the .ind files they represent and guessing at the longest prefix I can use to isolate the files I want to ignore. I suppose it's possible that I'd get it wrong on the last character or two, e.g., if I really just want `6_temp_wqp_fetch/out/` but instead I'm accidentally picking `6_temp_wqp_fetch/out/te` because of the example set of files I'm looking at...so it's still sorta approximate in that sense. But it seems better than manually wading through all the files every time, I hope.

Reply:
I think I _thought_ it didn't work, but now that I just went through the exercise of cleaning up all of those files, I think it was just ignoring a small subset. Since I was dealing w/ 1K+ files that were untracked, I didn't notice a difference and gave up on the approach of matching patterns across the bigger batch of files. 

I also would expect this to work.

Reply:
Cool, that's good info. It's still a pain, and we should definitely address it at some point (maybe at transition from FY18 to FY19 project work?)